logstash/tests/test_logstash.py

```diff
@@ -130,7 +130,7 @@ def test_check(aggregator):
             m_tags = m_tags + filter_tag
         if desc[0] == "gauge":
             aggregator.assert_metric(
-                m_name, tags=m_tags, count=1)
+                m_name, tags=m_tags, count=0)
```

Comment:
That's the proper fix. You need to change the tags, they know include the plugin_conf_id.